tox.ini

```diff
@@ -361,11 +361,11 @@ whitelist_externals =
     bash.exe
 commands =
     bash.exe -c 'mkdir -p cover-results'
-    bash.exe -c 'C:/Users/circleci/project/.tox/wincovercircle/Scripts/python -m coverage combine C:/Users/circleci/project/.tox/py37/.coverage*'
-    bash.exe -c 'C:/Users/circleci/project/.tox/wincovercircle/Scripts/python -m coverage xml'
+    bash.exe -c '~/project/.tox/wincovercircle/Scripts/python.exe -m coverage combine ~/project/.tox/py39/.coverage*'
```

Comment:
with the latest orb it's dynamic, something like `/c/Users/circleci.PACKER-62C74123`